e2e/explorer/api/src/app/etherscan/account.ts

```diff
@@ -59,3 +59,26 @@ export function txList(params: GetParams<TxList>): TransactionReceipt[] {
   const offset = Math.min(params.offset, 10000);
   return sorted.slice(offset*(page-1), offset*page);
 }
+
+/**
+ * return the list of blocks mined by an address
+ */
```

Comment:
you can inline it if your comment is on one line: 
```suggestion
/** Return the list of blocks mined by an address */
```